test/uk/gov/hmrc/exports/services/CountriesServiceSpec.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.services
+
+import org.scalatestplus.play.PlaySpec
+import uk.gov.hmrc.exports.models.Country
+
+class CountriesServiceSpec extends PlaySpec {
```

Comment:
Why we have to get whole application for this spec ?

Reply:
It doesnt @bambuchaAdm

Reply:
This has been addressed.

Reply:
I have addressed this